### DISCUSSION TITLE
menu local

### DISCUSSION THREAD

**Discussion by HCIE13821** [cobbler/cobbler#2962] [Version specific questions]
I installed cobbler 3.3.1, but after I imported the image, the menu only has the local option.

**Comment by SchoolGuy** [comment]
@HCIE13821 Do you use GRUB, iPXE or `pxelinux.0` for booting? is it identical for all three? Did you run a `cobbler sync`? Does the log contain any related error messages?

**Reply by HCIE13821** [reply]
使用的GRUB

**Reply by SchoolGuy** [reply]
Okay I hope Google Translate was correct.

Since you are using GRUB: Do you find at `/var/lib/cobbler/loaders` the `grub.efi` files?
